### PR TITLE
Post: announcing versioning schemes

### PR DESCRIPTION
**Pull request by stsewd**
ref:

- https://github.com/readthedocs/readthedocs.org/pull/10845
- https://github.com/readthedocs/readthedocs.org/pull/10766

### CODE REVIEW

File: introducing-versioning-schemes.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.. post:: November 28, 2023
+   :tags: docs, versions
+   :category: Changelog
+   :author: Santos
+   :location: CUE
+
+Introducing versioning schemes
+==============================
```
Comment (ericholscher) [inline introducing-versioning-schemes.rst:8]:
```suggestion
Introducing versioning schemes
==============================
```
```suggestion
Introducing support for version-only projects
=============================================
```

File: introducing-versioning-schemes.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  Resulting in URLs like:
+
+  - ``/index.html``
+  - ``/api/v3.html``
+
+While not all projects have or require translations, many benefit from having multiple versions.
+This meant that projects without translations but with need for multiple versions,
+had to opt into supporting translations in order to have multiple versions.
+
+We're excited to announce that we now support projects with multiple versions without translations.
```
Comment (ericholscher) [inline introducing-versioning-schemes.rst:33]:
```suggestion
**We're excited to announce that we now support projects with multiple versions without translations.**
```

**Review by ericholscher — APPROVED** [review]

File: introducing-versioning-schemes.rst (hunk 1 above)
Comment (stsewd) [reply introducing-versioning-schemes.rst:8]: what about "Introducing support for projects with multiple versions without translations"? It's a long title, not sure which one is better p:

Comment (ericholscher) [reply introducing-versioning-schemes.rst:8]: I think we can keep it as suggested -- it's the thing that's changing, so seems good to highlight. 👍

**Pull request merged**